src/components/ChatEntry.js

```diff
@@ -1,22 +1,53 @@
-import React from 'react';
+import { React, useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ sender, body, liked, timeStamp, id, onUpdateMessage }) => {
+  const [isLiked, setIsLiked] = useState(liked);
+
+  if (isLiked) {
+    console.log('is liked');
+  } else {
+    console.log('is unliked');
+  }
```

Comment:
Don't forget to get rid of debugging comments once you're ready to submit the final submission

```suggestion
```